Client/mods/deathmatch/logic/CClientIFP.h

```diff
@@ -0,0 +1,285 @@
+#ifndef __CCLIENTIFP_H
```

Comment:
File is missing the copyright/etc. comment at the top.
Also, use **#pragma once**

Reply:
About **#pragma once**, I remember using it for the header files, but I was told to remove it. It does no harm, I'll add it back.